internal/bundler/bundler_test.go

```diff
@@ -1,19 +1,28 @@
 package bundler
 
 import (
+	"fmt"
 	"path"
+	"strings"
 	"testing"
 
 	"github.com/evanw/esbuild/internal/fs"
 	"github.com/evanw/esbuild/internal/logging"
 	"github.com/evanw/esbuild/internal/parser"
 	"github.com/evanw/esbuild/internal/printer"
 	"github.com/evanw/esbuild/internal/resolver"
+	"github.com/kylelemons/godebug/diff"
 )
 
 func assertEqual(t *testing.T, a interface{}, b interface{}) {
 	if a != b {
-		t.Fatalf("%s != %s", a, b)
+		stringA := fmt.Sprintf("%v", a)
+		stringB := fmt.Sprintf("%v", b)
+		if strings.Contains(stringA, "\n") {
+			t.Fatal(diff.Diff(stringA, stringB))
```

Comment:
I understand why you did this, but this is going to make it harder to update tests since I've been copying and pasting the output of the tests to accept the new output, and this interleaves the old and new output. Luckily it's easy to disable by just commenting out a few lines, so I don't think it's a big problem.

Really I should figure out some system of doing snapshot-style testing so that test output can be updated automatically...